torch/optim/chain_lr_scheduler.py

```diff
@@ -0,0 +1,165 @@
+import math
```

Comment:
Since there is a new module, you will want to add a module-level docblock, and an entry for the module in the docs, in `docs/source/optim.rst`